src/components/BanDetail.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <v-list>
+        <v-list-item>
+          <v-list-item-icon>
+            <v-icon color="#8492e8">help</v-icon>
+          </v-list-item-icon>
+          <v-list-item-content>
+            <b>Active</b>
+            {{ban.active}}
+          </v-list-item-content>
+        </v-list-item>
+        <v-list-item>
+          <v-list-item-icon>
+            <v-icon color="grey">dns</v-icon>
+          </v-list-item-icon>
+          <v-list-item-content>
+            <b>Server</b>
+            {{ban.server}}
+          </v-list-item-content>
+        </v-list-item>
+        <v-list-item>
+          <v-list-item-icon>
+            <v-icon color="#6e9160">schedule</v-icon>
+          </v-list-item-icon>
+          <v-list-item-content>
+            <b>Created</b>
+            {{ban.start}}
+          </v-list-item-content>
+        </v-list-item>
+        <v-list-item>
+          <v-list-item-icon>
+            <v-icon color="#a8636b">schedule</v-icon>
+          </v-list-item-icon>
+          <v-list-item-content>
+            <b>Expires</b>
+            {{ban.end}}
+          </v-list-item-content>
+        </v-list-item>
+    </v-list>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'BanDetail',
```

Comment:
Can we pass in the invidual fields instead of one big object? So a prop for `server`, one for `banStart` etc.? This way we can also make use of the prop validation.